php-practice/section12.php

```diff
@@ -87,9 +109,13 @@ public function someMethod() {
  * Try to make a new SomeAbstractClass object and see what happens.
  * Explain in your own words, why did that happen?
  *
- * ANSWER HERE:
+ * ANSWER HERE: When I try to call someMethod I get the error "Fatal error: Cannot instantiate abstract class SomeAbstractClass". The reason is
+ * as you stated I can't "instantiate" the someMethod class. So that means I can't call it using echo $someAbstractObject->someMethod();
```

Comment:
In case the word "instantiate" feels weird, it just literally means "make an instance of".

Instance, hence instantiate.